// @sentry/core/dist/index.js
Object.defineProperty(exports, "__esModule", { value: true });
exports.Integrations = void 0;
var minimal_1 = require("@sentry/minimal");
Object.defineProperty(exports, "addBreadcrumb", { enumerable: true, get: function () { return minimal_1.addBreadcrumb; } });
Object.defineProperty(exports, "captureException", { enumerable: true, get: function () { return minimal_1.captureException; } });
Object.defineProperty(exports, "captureEvent", { enumerable: true, get: function () { return minimal_1.captureEvent; } });
Object.defineProperty(exports, "captureMessage", { enumerable: true, get: function () { return minimal_1.captureMessage; } });
Object.defineProperty(exports, "configureScope", { enumerable: true, get: function () { return minimal_1.configureScope; } });
Object.defineProperty(exports, "startTransaction", { enumerable: true, get: function () { return minimal_1.startTransaction; } });
Object.defineProperty(exports, "setContext", { enumerable: true, get: function () { return minimal_1.setContext; } });
Object.defineProperty(exports, "setExtra", { enumerable: true, get: function () { return minimal_1.setExtra; } });
Object.defineProperty(exports, "setExtras", { enumerable: true, get: function () { return minimal_1.setExtras; } });
Object.defineProperty(exports, "setTag", { enumerable: true, get: function () { return minimal_1.setTag; } });
Object.defineProperty(exports, "setTags", { enumerable: true, get: function () { return minimal_1.setTags; } });
Object.defineProperty(exports, "setUser", { enumerable: true, get: function () { return minimal_1.setUser; } });
Object.defineProperty(exports, "withScope", { enumerable: true, get: function () { return minimal_1.withScope; } });
var hub_1 = require("@sentry/hub");
Object.defineProperty(exports, "addGlobalEventProcessor", { enumerable: true, get: function () { return hub_1.addGlobalEventProcessor; } });
Object.defineProperty(exports, "getCurrentHub", { enumerable: true, get: function () { return hub_1.getCurrentHub; } });
Object.defineProperty(exports, "getHubFromCarrier", { enumerable: true, get: function () { return hub_1.getHubFromCarrier; } });
Object.defineProperty(exports, "Hub", { enumerable: true, get: function () { return hub_1.Hub; } });
Object.defineProperty(exports, "makeMain", { enumerable: true, get: function () { return hub_1.makeMain; } });
Object.defineProperty(exports, "Scope", { enumerable: true, get: function () { return hub_1.Scope; } });
var api_1 = require("./api");
Object.defineProperty(exports, "API", { enumerable: true, get: function () { return api_1.API; } });
var baseclient_1 = require("./baseclient");
Object.defineProperty(exports, "BaseClient", { enumerable: true, get: function () { return baseclient_1.BaseClient; } });
var basebackend_1 = require("./basebackend");
Object.defineProperty(exports, "BaseBackend", { enumerable: true, get: function () { return basebackend_1.BaseBackend; } });
var request_1 = require("./request");
Object.defineProperty(exports, "eventToSentryRequest", { enumerable: true, get: function () { return request_1.eventToSentryRequest; } });
var sdk_1 = require("./sdk");
Object.defineProperty(exports, "initAndBind", { enumerable: true, get: function () { return sdk_1.initAndBind; } });
var noop_1 = require("./transports/noop");
Object.defineProperty(exports, "NoopTransport", { enumerable: true, get: function () { return noop_1.NoopTransport; } });
var Integrations = require("./integrations");
exports.Integrations = Integrations;
