// @sentry/apm/dist/index.js
Object.defineProperty(exports, "__esModule", { value: true });
exports.addExtensionMethods = exports.Integrations = void 0;
var hubextensions_1 = require("./hubextensions");
Object.defineProperty(exports, "addExtensionMethods", { enumerable: true, get: function () { return hubextensions_1.addExtensionMethods; } });
var ApmIntegrations = require("./integrations");
exports.Integrations = ApmIntegrations;
var span_1 = require("./span");
Object.defineProperty(exports, "Span", { enumerable: true, get: function () { return span_1.Span; } });
Object.defineProperty(exports, "TRACEPARENT_REGEXP", { enumerable: true, get: function () { return span_1.TRACEPARENT_REGEXP; } });
var transaction_1 = require("./transaction");
Object.defineProperty(exports, "Transaction", { enumerable: true, get: function () { return transaction_1.Transaction; } });
var spanstatus_1 = require("./spanstatus");
Object.defineProperty(exports, "SpanStatus", { enumerable: true, get: function () { return spanstatus_1.SpanStatus; } });
// We are patching the global object with our hub extension methods
hubextensions_1.addExtensionMethods();
